client/ayon_blender/plugins/publish/extract_blend.py

```diff
@@ -115,31 +117,9 @@ def add_datablock(self, instance: pyblish.api.Instance) -> set:
         Returns:
             set: A set of data blocks added.
         """
-        data_blocks = set()
-
-        for data in instance:
-            data_blocks.add(data)
-            # Pack used images in the blend files.
-            if not (
-                isinstance(data, bpy.types.Object) and data.type == 'MESH'
-            ):
-                continue
-            for material_slot in data.material_slots:
-                mat = material_slot.material
-                if not (mat and mat.use_nodes):
-                    continue
-                tree = mat.node_tree
-                if tree.type != 'SHADER':
-                    continue
-                for node in tree.nodes:
-                    if node.bl_idname != 'ShaderNodeTexImage':
-                        continue
-                    # Check if image is not packed already
-                    # and pack it if not.
-                    if node.image and node.image.packed_file is None:
-                        node.image.pack()
-        return data_blocks
-
+        return {
+            data for data in instance
+        }
```

Comment:
I think we can now remove this method and just do `data_blocks = set(instance)` above?